java/src/main/java/org/rocksdb/AdvancedMutableColumnFamilyOptionsInterface.java

```diff
@@ -80,6 +80,25 @@ T setMemtablePrefixBloomSizeRatio(
    */
   double memtablePrefixBloomSizeRatio();
 
+  /**
+   * Enable whole key bloom filter in memtable. Note this will only take effect
+   * if memtable_prefix_bloom_size_ratio is not 0. Enabling whole key filtering
+   * can potentially reduce CPU usage for point-look-ups.
+   *
+   * Default: false (disable)
+   *
+   * @param memtableWholeKeyFiltering the memtable whole key filtering mode
```

Comment:
I think it's better to put "true iff whole key bloom filter is enabled in memtable" - it's not really a mode.